scripts/full_orga_sql_dump.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# ./scripts/full_orga_sql_dump.sh 1
+
+ORGANISATION_ID=$1
+DUMP_ID=$RANDOM
```

Comment:
```suggestion
DUMP_ID=$2
```

I had to change that as `$RANDOM` was not set.

Reply:
oh ? `echo $RANDOM` outputs a random number on my machine 🤔 I'm really bad with unix/bash so I don't know